packages/material-ui/src/Dialog/Dialog.js

```diff
@@ -21,7 +21,6 @@ export const styles = theme => ({
     flexDirection: 'column',
     flex: '0 1 auto',
     position: 'relative',
-    maxHeight: '90vh',
```

Comment:
Why did you removed this line?
It does not display properly with a big list like here: https://codesandbox.io/s/v033qjxyz7

Reply:
#11914

Reply:
Ok thanks.